common/helpers.go

```diff
@@ -46,6 +46,26 @@ func initEnviron() {
 	}
 }
 
+// CleanURL is similar to path.Clean() but to work on URLs
```

Comment:
Can we add a small summary then of what it does? Otherwise I need to hunt down path.Clean() and read their docstring.